.github/workflows/release.yml

```
@@ -140,11 +160,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
         with:
-          ref: ${{ needs.verify-inputs.outputs.RELEASE_BRANCH }}
+          ref: ${{ needs.verify-inputs.outputs.WORKING_BRANCH }}
 
       - name: Update enterprise image version
         run: |
-          sed -i "s/defaultImage = \"v[0-9]\+\.[0-9]\+\.[0-9]\+\"/defaultImage = \"${VERSION}\"/" internal/config/image.go
+          sed -i "s/defaultImage = \"[^\"]*\"/defaultImage = \"${VERSION}\"/" internal/config/image.go
```

I know you didn't add the command here, but maybe a good opportunity to make it more robust?
```suggestion
          defaultVersionReg='defaultImage = \"[^\"]*\"'

          # Ensure regexp matches (otherwise the file was changed or the workflow is broken).
          grep -E "${defaultVersionReg}" internal/config/image.go

          sed -i "s/${defaultVersionReg}/defaultImage = \"${VERSION}\"/" internal/config/image.go
```